### PR TITLE
rename loggedInEmail to loggedInUser with backcompat support.  issue #18...

### PR DESCRIPTION
**Pull request by lloyd**
...05

### CODE REVIEW

File: resources/static/include_js/include.js [hunk 1]
```diff
@@ -1027,13 +1027,25 @@
...
+      if (typeof options.loggedInEmail !== 'undefined' &&
+          typeof options.loggedInUser !== 'undefined')
+      {
+        throw "you cannot supply *both* loggedInEmail and loggedInUser";
+      }
+      else if(typeof options.loggedInEmail !== 'undefined')
+      {
+        options.loggedInUser = options.loggedInEmail;
+        delete options.loggedInEmail;
+      }
```
Comment (callahad) [inline resources/static/include_js/include.js:1044]: Looks good! Pedantry -- shouldn't this be using same-line curly braces to match the rest of this file?

**Comment by callahad** [comment]
Should we add a unit test for this? Otherwise, r+.

**Comment by benadida** [comment]
@callahad is right, we need tests, I guess this means we wait until early next week to merge, that's probably ok.

**Comment by lloyd** [comment]
oh damn.  This means we can't really call the .watch() api stable when we roll 2012.06.22 on july 4th.

**Pull request merged**